### PR TITLE
Fix issues when showing plots in examples

### LINKED ISSUE #1657: How is voltage_trace supposed to be shown?
**Describe the flaw**
In this commit 87edd9e4a10aa3524be223a499b03293ac34a579 the show method was removed. It is still included in the documentation in the examples, putting the examples in a non-working state, and making me wonder how to plot the voltages.

Examples:
https://nest-simulator.readthedocs.io/en/latest/auto_examples/one_neuron.html
https://nest-simulator.readthedocs.io/en/latest/auto_examples/twoneurons.html

**Suggest an improvement**
Replace code how the voltage is supposed to be plotted with a working version.

### PR DESCRIPTION
**Pull request by BSVogler**
Fixes the examples by showing how to plot the generated plot.

### CODE REVIEW

**Comment by jougs** [comment]
Thanks for you contribution!

One general question before going into a deeper review: do you see a problem with this work when running the examples on a headless system (like, for instance, Travis CI) or would that cause a problem?

Also, this PR does seem to be incomplete. I stumbled over at least [brunel_alpha_nest.py](https://github.com/nest/nest-simulator/blob/master/pynest/examples/brunel_alpha_nest.py), which uses the `raster_plot` module and tries to call it's long-gone `show()` method. Would you be able to check all occurrences and also fix them within this PR?

**Comment by jougs** [comment]
BTW: pulling current upstream master should fix the build issues on Travis.

**Comment by BSVogler** [comment]
I can take a look at the raster_plot module.

> BTW: pulling current upstream master should fix the build issues on Travis.
I don't understand what you mean. I pulled my branch before adding this pull request.

**Comment by BSVogler** [comment]
I replaced all `raster_plot.show()` calls.

**Comment by jougs** [comment]
@BSVogler: It seems that the failure on Travis is indeed not caused by you not properly merging master, but by them (again) updating Python on their macOS VMs. I'm currently testing a fix and will create an emergency PR later today. Once that is merged, you should pull again and all should be good.

Regarding the policy for whether to include `show()` in the examples at all, we came to the following conclusion in [yesterday's Open NEST Developer Video Conference](https://github.com/nest/nest-simulator/wiki/2020-07-20-Open-NEST-Developer-Video-Conference):
* We will accept this PR so that users experience the least confusion and get plots directly when running the examples
* the documentation can stay as is
* for our build pipelines ran on headless systems (i.e. CI on Travis and documentation extraction/generation on Read The Docs), we will monkey patch the examples to either suppress figures altogether or write them to files instead of trying to show them.

**Comment by jougs** [comment]
Here's the PR fixing the build issues is #1700

File: pynest/examples/brunel_alpha_nest.py [hunk 1]
```diff
@@ -99,7 +100,7 @@ def ComputePSPnorm(tauMem, CMem, tauSyn):
 # Assigning the simulation parameters to variables.
 
 dt = 0.1    # the resolution in ms
-simtime = 1000.0  # Simulation time in ms
+simtime = 100.0  # Simulation time in ms
```
Comment (jougs) [inline pynest/examples/brunel_alpha_nest.py:103]: This change seems unrelated. Please remove (or provide an explanation why it should be here).

**Comment by jougs** [comment]
@BSVogler: #1700 was merged. Please `git pull upstream master`.

File: pynest/examples/brunel_alpha_nest.py (hunk 1 above)
Comment (BSVogler) [reply pynest/examples/brunel_alpha_nest.py:103]: yes, this was added in a mistake.

**Comment by jougs** [comment]
#1623 fixes another build issue on Travis. Please `git pull upstream master` again. Sorry for the inconveniences.

**Comment by sarakonradi** [comment]
@BSVogler Thanks for fixing the plot-related issue! 

Could either you or @jougs mention #1657 in the PR description? That would be fantastic.

**Comment by jougs** [comment]
@sarakonradi: Actually it is sufficient if the magic word is mentioned in a comment (also one by you). Like so: fixes #1657.
